.github/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+pkgs/args           @munificent
+pkgs/async          @dart-ecosystem-admins
+pkgs/characters     @dart-ecosystem-admins
+pkgs/collection     @dart-ecosystem-admins
+pkgs/convert        @dart-ecosystem-admins
+pkgs/crypto         @dart-ecosystem-admins
+pkgs/fixnum         @lrhn
```

Comment:
Thanks. It's not really _my_ package. (I do want to rewrite it, but so far I haven't really touched it. I don't know who knows better.)

The `characters` and `os_detect` packages, on the other hand, are pretty much all my code.

The `async`, `collection`, `convert` and `typed_data` packages are companions to the platform libraries, and should have (mostly) the same owners. That might be `@dart-language-team`, but I'm not sure everybody there would care.
Maybe we should have a `@dart-core-libraries-team`. I volunteer myself and @natebosch.

Reply:
> Maybe we should have a `@dart-core-libraries-team`. I volunteer myself and @natebosch.

SGTM